README.md

```diff
@@ -3,161 +3,163 @@
 
 ## Workflow
 
-Istio Auth workflow consists of two phases, deployment and runtime. We briefly
-cover each phase in this section and a more detailed version can be found
-[here](https://docs.google.com/document/d/1spoQ9MIb7ABFDdFzlFITczCbH_AHO3RXSgLLeXAYIJU/edit).
-
-### Deployment Phase
-
-1. Istio CA watches K8s API Server, creates a
-   [SPIFFE](https://spiffe.io/docs/svid/) cert/key pair for each of the
-   existing and new service accounts, and sends them to API Server.
-2. When a pod is created, API Server mounts the cert/key according to the
-   service account using [Kubernetes
-   secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
-3. [Istio-Pilot](https://github.com/istio/pilot/blob/master/doc/design.md)
-   generates the config with proper cert/key and secure naming information,
-   which defines what service account(s) can run a certain service, and passes it
-   to Envoy.
-
-### Runtime Phase
-
-1. The outbound traffic from a client service is rerouted to its local Envoy.
-2. The client side Envoy starts mutual TLS handshake with the server side
-   Envoy. During the handshake, it also does secure naming check to verify that
-   the service account presented in the server certificate can run the server
-    service.
-3. The traffic is forwarded to the server side Envoy after mTLS connection is
-   established, which is then forwarded to the server service through local TCP
-   connections.
-
-## Service to Service Auth Best Practice
-
-In this section, we provide a few deployment guidelines and then discuss a
-real-world scenario.
-
-### Deployment Guidelines
-
-- If there are multiple service operators (a.k.a.
-  [SREs](https://en.wikipedia.org/wiki/Site_reliability_engineering) deploying
-  different services in a cluster (typically in a medium- or large-size
-  cluster), we recommend creating a [separate
-  namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/)
-  for each SRE team to isolate their access. For example, we can create a
-  “team1-ns” namespace for team1, and “team2-ns” namespace for team2, such that
-  both teams won’t be able to access each other’s services.
-- If Istio CA is compromised, all its managed key & cert in the cluster may be
-  exposed. We strongly recommend to run Istio CA on a dedicated namespace
-  (e.g., istio-ca-ns) which only cluster admins have access to.
-  - We are looking into running Isito CA in the kubernetes master in the future releases.
+The Istio Auth workflow consists of two phases, deployment and runtime. For the deployment phase, we discuss the two scenarios (i.e., in Kubernetes and VM/bare-metal machines) separately since they are different. Once the key and certificate are deployed, the runtime phase is the same for the two scenarios. We briefly cover the workflow in this section.
+
+### Deployment phase (Kubernetes Scenario)
+
```

Comment:
Do we need multiple empty lines (also below)?

Reply:
Removed.